src/http/AuthHeader.js

```diff
@@ -96,6 +97,17 @@ class AuthHeader {
         }
         return config.KeyLevel.LOW;
     }
+
+    static _customerInitiated(context) {
+        // if the customer initiated request flag is set to true, we add it to the header, and reset the flag.
+        if (context !== undefined &&
+                context.customerInitiated !== undefined &&
+                context.customerInitiated === true) {
```

Comment:
seems redundant
`if (context && context.customerInitiated)` ?

Reply:
I'm not sure how js works. It won't scream at me if context.customerInitiated is undefined?

Reply:
if context.customerInitiated is undefined, the `if` won't go through. `undefined` evaluates to `false`

Reply:
ok, changed.